units/systemd-syscfg.service

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
this service file should be added in a separate commit *after* the tool itself has been added, as this then hooks it into the system.

in fact, the documentation should also be added after the tool, usually.